lemur/common/validators.py

```diff
@@ -152,6 +152,18 @@ def dates(data):
                         data["authority"].authority_certificate.not_after
                     )
                 )
+            # Allow no more than PUBLIC_CA_MAX_VALIDITY_DAYS (Default: 397) days of validity
+            # for certs issued by public CA
+            # The list of public issuers can be managed through a config named PUBLIC_CA
+            public_CA = current_app.config.get("PUBLIC_CA", [])
```

Comment:
maybe `PUBLIC_CA_AUTHORITY_NAMES` to indicate what this config is expected to contain, and that it is a list.

Reply:
Done